components/elm/src/biogeochem/AllocationMod.F90

```diff
@@ -106,11 +106,9 @@ module AllocationMod
   !$acc declare create(col_plant_pdemand(:))
 
   logical :: crop_supln  = .false.    !Prognostic crop receives supplemental Nitrogen
-
-  real(r8), allocatable :: decompmicc(:,:) ! column-level soil microbial decomposer biomass gC/m3
-
-  real(r8), parameter   :: E_plant_scalar  = 0.0000125_r8 ! scaling factor for plant fine root biomass to calculate nutrient carrier enzyme abundance
-  real(r8), parameter   :: E_decomp_scalar = 0.05_r8      ! scaling factor for plant fine root biomass to calculate nutrient carrier enzyme abundance
+  real(r8), allocatable :: decompmicc(:,:)                ! column-level soil microbial decomposer biomass gC/m3
```

Comment:
`decompmicc`
  - In `master` is 1D for `j`-level
  - In this branch is 2D for `c,j`
Question for @qzhu-lbl @rgknox: Should this be 1D or 2D?

Reply:
asfaict (as far as I can tell, "as far as I can tell" is not yet an acronym in the public lexicon, so I will define it here, asfaict = as far as I can tell) there is no reason to make this 2D, because we do not need to remember the values from other columns as we make calculations on the column of interest.  We also don't output this in history, so allocating just on soil depth is cheaper and more efficient.